transformer.py

```
@@ -0,0 +1,113 @@
+full_dataset_subset = full_dataset.select(range(num_samples_to_use))
+
+print("로드된 데이터셋 정보:")
+print(full_dataset_subset)
+
+print("\n첫 번째 데이터 예시 (text 확인):")
+print(full_dataset_subset[0]['text'])
+
+# 데이터셋 확인을 위해 Pandas DataFrame으로 변환 (선택 사항)
+df_check = pd.DataFrame(full_dataset_subset)
+print("\n데이터셋 일부 미리보기 (DataFrame):")
+display(df_check.head(3)) # Colab 환경에서는 display()가 표 형태로 보여줍니다
```

Ruff (0.13.2)

Undefined name `display`

(F821)